tests/test_p4_exception_handling.py

```diff
@@ -91,12 +91,8 @@ def test_p4_print_exception_in_sync(perforce_environment: P4TestEnvironment, std
     stdout_checker.assert_has_calls_with_param(text)
 
 
-def test_p4_print_exception_wrong_shelve(perforce_environment: P4TestEnvironment, stdout_checker: FuzzyCallChecker):
+def test_p4_print_exception_already_committed_shelve(perforce_environment: P4TestEnvironment, stdout_checker: FuzzyCallChecker):
     cl = perforce_environment.vcs_client.make_a_change()
     perforce_environment.settings.PerforceMainVcs.shelve_cls = [cl]
-
-    # This is not the 'already committed' case of Swarm review, so it actually should fail
     perforce_environment.run(expect_failure=True)
-    stdout_checker.assert_has_calls_with_param(
-        f"Errors during command execution( \"p4 unshelve -s {cl} -f\" )")
-    stdout_checker.assert_has_calls_with_param(f"[Error]: 'Change {cl} is already committed.'")
+    stdout_checker.assert_has_calls_with_param(f"Change {cl} is already committed.")
```

Comment:
Why there is no "Errors during command execution" log?
Is this some change in P4Python?

Reply:
Yep, changes in [p4python 2024.1.0](https://github.com/perforce/p4python/compare/2023.2.2...2024.1.0).